mne/io/cnt/cnt.py

```diff
@@ -0,0 +1,230 @@
+        n_channels = np.fromfile(fid, dtype='<u2', count=1)[0]
+
+        data_offset = n_channels * start * n_bytes + offset
+        fid.seek(376)
+        cnt_info['sfreq'] = np.fromfile(fid, dtype='<u2', count=1)[0]
+        fid.seek(438)
+        lowpass_toggle = np.fromfile(fid, 'i1', count=1)[0]
+        highpass_toggle = np.fromfile(fid, 'i1', count=1)[0]
+        fid.seek(869)
+        lowcutoff = np.fromfile(fid, dtype='f4', count=1)[0]
+        fid.seek(2, 1)
+        highcutoff = np.fromfile(fid, dtype='f4', count=1)[0]
+
+        fid.seek(886)
+        cnt_info['event_offset'] = np.fromfile(fid, dtype='<i4', count=1)[0]
+        cnt_info['continuous_seconds'] = np.fromfile(fid, dtype='<f4',
+                                                     count=1)[0]
+        cnt_info['channel_offset'] = np.fromfile(fid, dtype='<i4', count=1)[0]
+        if cnt_info['channel_offset'] > 1 and read_blocks:
+            cnt_info['channel_offset'] /= 2
+            warn('Reading in data in blocks of %d. If this fails, try using '
+                 'read_blocks=False.')
+        cnt_info['n_samples'] = (cnt_info['event_offset'] -
+                                 (900 + 75 * n_channels)) / (2 * n_channels)
+        ch_names = list()
+        cals = list()
+        baselines = list()  # Baselines are subtracted before scaling the data.
+        chs = list()
+        for i in range(n_channels):
+            fid.seek(data_offset + 75 * i)
+            ch_names.append(''.join(np.fromfile(fid, dtype='S1', count=10)))
+            fid.seek(data_offset + 75 * i + 47)
+            baselines.append(np.fromfile(fid, dtype='i2', count=1)[0])
+            fid.seek(data_offset + 75 * i + 59)
+            sensitivity = np.fromfile(fid, dtype='f4', count=1)[0]
+            fid.seek(data_offset + 75 * i + 71)
+            cal = np.fromfile(fid, dtype='f4', count=1)
+            cals.append(cal * sensitivity * 1e-6 / 204.8)
+    cnt_info['n_channels'] = n_channels
+
+    info = _empty_info(cnt_info['sfreq'])
+    if lowpass_toggle == 1:
+        info['lowpass'] = highcutoff
+    if highpass_toggle == 1:
+        info['highpass'] = lowcutoff
+    info.update(filename=input_fname,
+                meas_date=calendar.timegm(date.utctimetuple()),
+                description=None, buffer_size_sec=10.)
+    for idx, ch_name in enumerate(ch_names):
+        ch_coil = FIFF.FIFFV_COIL_EEG
+        ch_kind = FIFF.FIFFV_EEG_CH
+        chan_info = {'cal': cals[idx], 'logno': idx + 1, 'scanno': idx + 1,
+                     'range': 1.0, 'unit_mul': 0., 'ch_name': ch_name,
+                     'unit': FIFF.FIFF_UNIT_V,
+                     'coord_frame': FIFF.FIFFV_COORD_HEAD, 'loc': np.zeros(12),
+                     'coil_type': ch_coil, 'kind': ch_kind}
+        chs.append(chan_info)
+
+    # Add the stim channel.
+    chan_info = {'cal': 1.0, 'logno': len(chs) + 1, 'scanno': len(chs) + 1,
+                 'range': 1.0, 'unit_mul': 0., 'ch_name': 'STI',
```

Comment:
We usually hard-code the one that we synthesize, but we typically call it `STI 014` to use the old standard name so this is a typo